devel/addon/certmanager/install.sh

```diff
@@ -53,10 +54,8 @@ kubectl get namespace "${NAMESPACE}" || kubectl create namespace "${NAMESPACE}"
 # Build the Helm chart package .tgz
 bazel build //deploy/charts/cert-manager
 
-# Pre-compile the kubectl plugin, so it can quickly check the api status
-bazel build //hack/bin:kubectl-cert_manager
-
 # Upgrade or install cert-manager
+# --wait flag should wait for Job to complete
```

Comment:
Does it? I thought we needed to also supply `--wait-for-jobs`.

```
helm upgrade --help
...
--wait-for-jobs                if set and --wait enabled, will wait until all Jobs have been completed before marking the release as successful. It will wait for as long as --timeout
```

Reply:
Indeed, I'll update
FYI ref code: https://github.com/helm/helm/blob/281380f31ccb8eb0c86c84daf8bcbbd2f82dc820/pkg/kube/ready.go#L106-L111